docs/source/kinds_of_types.rst

```diff
@@ -146,8 +146,30 @@ purpose. Example:
 
 .. _callable-types:
 
-Callable types (and lambdas)
-****************************
+Callable types
+**************
+
+A callable type is a type that conforms to the ``collections.abc.Callable`` protocol, i.e. it has a ``__call__`` attribute (invoked with the call operator: ``X()``).
+There are three main kinds of these:
+
+- functions (and lambdas)
```

Comment:
mention the type

Reply:
> - functions and lambdas (``types.FunctionType``/``builtins.function``)

hmm, does this now look a bit like it's saying functions are `types.FunctionType` and lambdas are `builtins.function`?